### PR TITLE
Merge contributing docs

### PR DESCRIPTION
**Pull request by mlvandijk**
Merge and update several contributing files to include information on how to contribute to the documentation.

### CODE REVIEW

**Comment by mlvandijk** [comment]
Build fails due to external links in about/blog-posts.md;
not related to changes in this PR!

File: content/about/contributing-to-documentation.md [hunk 1]
```diff
@@ -111,3 +91,66 @@ In general, it should be brief and to the point.
   * `(platform-consistent)` means this works the same on all platforms, like Gherkin
   * `(platform-inconsistent)` means this currently works differently across platforms, like formatter outputs
     * (The Cucumber team is working to make the implementations more consistent, but this takes time)
+
+## Toolchain
+The documentation is written in [Markdown](http://toolchain.gitbook.com/syntax/markdown.html)
+(simple markup) and [AsciiDoc](http://toolchain.gitbook.com/syntax/asciidoc.html) (richer markup).
```
Comment (plaindocs) [inline content/about/contributing-to-documentation.md]: Aciidoc is no longer relevant here.

**Review by plaindocs — CHANGES_REQUESTED** [review]
Good stuff!

**Comment by mlvandijk** [comment]
@plaindocs Thanks! I removed the bit about asciidocs... (Update: and resolved a merge conflict)

**Pull request merged**